Source/Csla.AspNetCore/RazorPages/PageModel.cs

```diff
@@ -45,9 +46,10 @@ public async Task<bool> SaveAsync(bool forceUpdate = false)
       try
       {
         if (Item is BusinessBase bb && !bb.IsValid)
-          AddBrokenRuleInfo(Item, null);
+          AddBrokenRuleInfo(Item, string.Empty);
```

Comment:
I wonder if `string.Empty` and `null` render differently in html? I've never tested that but I know when a null is returned that the bound html element doesn't appear (or take up space in the UI). 

We should confirm that an empty string has the same behavior when rendered.

Reply:
Don't know about the html. And in this case it's irrelevant because the `AddBrokenRuleInfo` method won't use it. _But_ if it's used with `null` that will throw an `ArgumentNullException` because the `ModelState.AddModelError()` method does not accept `null` as an error message.
https://github.com/dotnet/aspnetcore/blob/ee4a0e9801c4e1e673d0373e1c51a6790de174cc/src/Mvc/Mvc.Abstractions/src/ModelBinding/ModelStateDictionary.cs#L324-L330